taxonomy-manager-engine/src/main/java/com/digirati/taxonomy/manager/lookup/ConceptExtractor.java

```diff
@@ -91,12 +90,11 @@ public void updateConcept(
             return;
         }
 
-        originalLabels.forEach(
-                label -> {
-                    Set<UUID> uuids = Sets.newHashSet(conceptLookupTable.get(label));
-                    uuids.remove(conceptUuid);
-                    conceptLookupTable.replaceValues(label, uuids);
-                });
+        originalLabels.forEach(label -> {
```

Comment:
For simple cases like this (simple case being a stream or collection where we only iterate over, and don't do any pipeline or terminal operations on). I think we should use the imperative for-each instead of streams. It's a little faster and easier to read.

What do you think?

Reply:
I agree in principle, but I'd rather use either streams or the imperative alternative consistently within a method. Since the stream version is definitely needed for building the set of original labels I went for this form, but I'm happy to pull the building of the set out into its own method so that this can be switched to an imperative for each if you feel strongly about this

Reply:
I'm not sure the argument of consistency within a method body holds when you're looking at the entire application.

Instead of being inconsistent with the rest of the method body, it's then inconsistent with the rest of the codebase where `Streams` aren't used at all (e.g. `deleteConcept(...)` below, which uses the imperative form and has no `Stream` references). 

I think it's better for an entire codebase to be consistent over consistency between statements in method body. Especially when the rules over code-style within a method are dependent on the code within it.

I don't feel too strongly about this, but it seems like we should be choosing the best tool for the job in every case, and it doesn't feel like we're doing that here.